pages/_next-prefetcher.js

```diff
@@ -0,0 +1,108 @@
+/* global self */
```

Comment:
I personally prefer consistency than small optimizations which we can improve later if required.